exa/core/box.py

```diff
@@ -31,30 +45,96 @@ def groupby(self):
         pass
 
     def info(self):
-        pass
+        info = []
+        for name, data in self._data.items():
+            df = data.data()
+            # TODO: prepend __module__?
+            if isinstance(df, pd.DataFrame):
+                info.append((name, type(data), df.memory_usage().sum(), df.shape))
+            elif isinstance(df, pd.Series):
+                info.append((name, type(data), df.memory_usage(), df.shape))
+            else:
+                info.append((name, type(data), None, None))
+        info = pd.DataFrame(info, columns=['name', 'type', 'size', 'shape'])
+        return info.set_index('name').sort_index()
 
     def memory(self):
         pass
 
     def network(self):
-        pass
+        """Build a network of the relationships between
+        data as specified by the container's metadata.
```

Comment:
I think the relationships are being built by the data itself (assumes that the data is `DataFrame` or `Series`? object and constructs based on series names).

Reply:
For the network I was hoping that it would be possible to generate some (if not all) of the relationships based solely on the Data's metadata, rather than having to descend to the underlying data and inspect it directly. I see this approach not working great where multiple index-column relations exist, e.g. atom -> atom0, atom1.